src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -1767,6 +1767,143 @@ public Task<bool> UnpinChatMessageAsync(ChatId chatId, CancellationToken cancell
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        /// <returns>Returns the uploaded File on success.</returns>
+        /// <see href="https://core.telegram.org/bots/api#uploadstickerfile"/>
+        public Task<File> UploadStickerFileAsync(int userId, FileToSend pngSticker, 
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var parameters = new Dictionary<string, object>()
+            {
+                { "user_id", userId },
+                { "png_sticker", pngSticker }
+            };
+
+            return SendWebRequestAsync<File>("uploadStickerFile", parameters, cancellationToken);
+        }
+
+        /// <summary>
+        /// Use this method to create new sticker set owned by a user. The bot will be able to edit the created sticker set. 
+        /// </summary>
+        /// <param name="userId">User identifier of created sticker set owner</param>
+        /// <param name="name">Short name of sticker set, to be used in t.me/addstickers/ URLs (e.g., animals). Can contain only english letters, digits and underscores. Must begin with a letter, can't contain consecutive underscores and must end in “_by_&lt;bot_username&gt;”. &lt;bot_username&gt; is case insensitive. 1-64 characters.</param>
+        /// <param name="title">Sticker set title, 1-64 characters</param>
+        /// <param name="pngSticker">Png image with the sticker, must be up to 512 kilobytes in size, dimensions must not exceed 512px, and either width or height must be exactly 512px.</param>
+        /// <param name="emojis">One or more emoji corresponding to the sticker</param>
+        /// <param name="isMasks">Pass True, if a set of mask stickers should be created</param>
+        /// <param name="maskPosition">Position where the mask should be placed on faces</param>
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        /// <returns>Returns True on success.</returns>
+        /// <see href="https://core.telegram.org/bots/api#createnewstickerset"/>
+        public Task<bool> CreateNewStickerSetAsnyc(int userId, string name, string title, FileToSend pngSticker,
+            string emojis, bool isMasks = false, MaskPosition maskPosition = null, 
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var parameters = new Dictionary<string, object>()
+            {
+                { "user_id", userId },
+                { "name", name },
+                { "title", title },
+                { "png_sticker", pngSticker },
+                { "emojis", emojis }
+            };
+
+            if (isMasks)
+                parameters.Add("is_masks", isMasks);
```

Comment:
There is a `contains_masks` property according to [Tg docs](https://core.telegram.org/bots/api#createnewstickerset). Is this a mistake in docs or here?

_Unfortunately, I cannot test it right now._